packages/makeswift/src/init.ts

```diff
@@ -64,7 +65,14 @@ async function init(
   const selectSiteUrl = new URL(`${MAKESWIFT_APP_ORIGIN}/cli/select-site`)
   selectSiteUrl.searchParams.set('project_name', projectName)
   selectSiteUrl.searchParams.set('callback_url', callbackUrl)
-  await open(selectSiteUrl.toString())
+
+  const selectSiteUrlString = selectSiteUrl.toString()
+  console.log(
+    `\nRedirecting you to ${chalk.blue(
```

Comment:
What is instead of "Redirecting" we said "opening your browser at"? "Redirecting" sounds weird since we're not doing a redirect here and are instead opening their browser

Reply:
Yeah sure